ToDo/src/components/Tasks/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {
+  FlatList,
+  KeyboardAvoidingView,
+  Platform,
+  StyleSheet,
+  View,
+} from 'react-native';
+import {useAppSelector} from '../../hooks/hooks';
+import Task from '../Task';
+
+function Tasks() {
+  const tasks = useAppSelector(state => state.tasks);
+
+  return (
+    <View style={styles.container}>
+      <KeyboardAvoidingView
+        keyboardVerticalOffset={Platform.OS === 'ios' ? 40 : 0}
```

Comment:
Let's also add `enabled` prop for iOS only. On android this works out of the box